hack/.packages

```diff
@@ -13,7 +14,6 @@ k8s.io/kops/dns-controller/pkg/util
 k8s.io/kops/dns-controller/pkg/watchers
 k8s.io/kops/examples/kops-api-example
 k8s.io/kops/federation
-k8s.io/kops/federation/model
```

Comment:
That was the result I got from running 
```
go list k8s.io/kops/... | grep -v vendor > hack/.packages"
```
and I did double check, that directory is not a go package https://github.com/kubernetes/kops/tree/master/federation/model